README.rst

```diff
@@ -303,7 +303,6 @@ Overview of Directories
 ``CMake``               contains CMake helper scripts
 ``config``              contains reusable default algorithm configuration files
 ``doc``                 contains release notes, manuals, and other documentation
-``examples``            contains example tool configuration for public datasets
```

Comment:
maybe we leave this, but say "contains pointers to example public datasets to use"